packages/app-mobile/components/screens/tags.js

```diff
@@ -83,7 +83,7 @@ class TagsScreenComponent extends BaseScreenComponent {
 	async componentDidMount() {
 		const tags = await Tag.allWithNotes();
 		tags.sort((a, b) => {
-			return a.title.toLowerCase() < b.title.toLowerCase() ? -1 : +1;
+			return a.title.localeCompare(b.title, undefined, { sensitivity: 'accent' });
```

Comment:
I don't really get it. You've created a sortTags function, but here you're manually sorting them again? Wasn't the idea to use the same sort function everywhere?

Reply:
For Desktop I had written the sortTags function in Tags.ts which was located in packages/lib/models (basically directly in the codebase). But in the case for android the Tag.ts file is located in an npm package i.e node_modules/@joplin/lib [https://www.npmjs.com/package/@joplin/lib](url). In order to add the sortTags function to Tags.ts in android I will have to make changes to the @joplin/lib package.

Reply:
It's the same package. To modify `@joplin/lib`, you change the files under packages/lib. It's all in BUILD.md, which you should read again, the lib package is shared by all apps

Reply:
Yep its the same package. This is something related to lerna. I will make the necessary changes and commit as soon as possible.

Reply:
Update: Firstly I used the `sortTags` function to sort tags in Android/iOS and it worked. The symlink to `@joplin/lib` in app-mobile was created on running yarn install in the root directory. Now the `@joplin/lib` Tags file contains the `sortTags` function but the package is also hosted on npm so will I have to publish the updated package to npm using lerna?

Reply:
I'm not familiar with any of this as I never deal with symlinks. In dev it should just use the files that are there, under /packages. It might help to start over from scratch the build by following BUILD.md.